web/src/main/java/se/inera/intyg/intygstjanst/web/service/dto/OccupationType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygstjanst.web.service.dto;
+
+public enum OccupationType {
+
+    OCCUPATION_NUVARANDE_ARBETE("Nuvarande arbete"),
```

Comment:
Redundant with OCCUPATION in the name for each OccupationType. Its enough with OccupationType.NUVARANDE_ARBETE instead of OccupationType.OCCUPATION_NUVARANDE_ARBETE.